### ISSUE TITLE
iOS xcodebuild -archive is broken in 1.8.3-dev

### ISSUE THREAD

**Issue by nate-eisner** [flutter/flutter#37482]
Looks like a repeat of this issue:
https://github.com/flutter/flutter/issues/22306

## Steps to Reproduce

This broke our CI builds after upgrading to 1.8.3-dev:
1.  `flutter build ios --flavor=prod -t lib/main-prod.dart --release --verbose --no-codesign`
2.  ` xcodebuild -sdk iphoneos -configuration Release-prod -workspace Runner.xcworkspace -scheme prod archive`
3. ???
4. Failure

## Logs

Then end of logs for the `xcodebuild archive`:
```
[command]/usr/libexec/PlistBuddy -c Print Name _xcodetasktmp.plist
Prod
[command]/bin/rm -f _xcodetasktmp.plist
[command]/usr/libexec/PlistBuddy -c Print CFBundleIdentifier /Users/vsts/agent/2.155.1/work/1/s/ios/build/prod.xcarchive/Products/Applications/Runner.app/Info.plist
com.ourcompany.mobile
[command]/usr/libexec/PlistBuddy -c Add provisioningProfiles:com.ourcompany.mobile string Prod _XcodeTaskExportOptions.plist
[command]/usr/bin/xcodebuild -exportArchive -archivePath /Users/vsts/agent/2.155.1/work/1/s/ios/build/prod.xcarchive -exportPath /Users/vsts/agent/2.155.1/work/1/s/ios/build -exportOptionsPlist _XcodeTaskExportOptions.plist
2019-08-02 15:39:45.547 xcodebuild[2946:18223] [MT] IDEDistribution: -[IDEDistributionLogging _createLoggingBundleAtPath:]: Created bundle at path '/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/prod_2019-08-02_15-39-45.546.xcdistributionlogs'.
error: exportArchive: Symbols tool failed

Error Domain=IDEFoundationErrorDomain Code=1 "Symbols tool failed" UserInfo={NSLocalizedDescription=Symbols tool failed}

** EXPORT FAILED **

##[error]Error: The process '/usr/bin/xcodebuild' failed with exit code 70
```

```
Running flutter doctor...
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel dev, v1.8.3, on Mac OS X 10.14.6 18G84, locale en)
[!] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses
[✓] Xcode - develop for iOS and macOS (Xcode 10.2.1)
[!] Android Studio (not installed)
[!] Connected device
    ! No devices available
```

**Comment by jmagman** [comment]
This should be fixed, see https://github.com/flutter/flutter/issues/37047.  Let us know if you are still seeing symbols tool failures.

**Comment by rmaalouf** [comment]
Getting the exact same error.


```
Flutter 1.8.3 • channel beta • git@github.com:flutter/flutter
Framework • revision e4ebcdf6f4 (13 days ago) • 2019-07-27 11:48:24 -0700
Engine • revision 38ac5f30a7
Tools • Dart 2.5.0 (build 2.5.0-dev.1.0 0ca1582afd
```

**Comment by jmagman** [comment]
It landed in 1.8.4 and you're on 1.8.3.  You'll have to wait a bit for it to reach the beta channel.
https://github.com/flutter/flutter/commit/a8fbe02a9d7038743148e1008220e7a500eb69c6.

**Comment by wald-tq** [comment]
It also works in the stable channel with version 1.7.8+hotfix.4
